### PR TITLE
fixed Typos/mistakes

### LINKED ISSUE #585: Adding a collapse section for all the contributors in the mentorship page
hey, @klumb I think this small enhancement can be helpful for the website, Please have a look!
Right now in the [mentorship section](https://chaoss.community/chaoss-mentorship/) of the website has a table listing all the contributors that have contributed to this organization through many programs, but the links to the individual contributor section in the table section will only work in Github markdown, but not in the website
For Example : 
> |[Ria Gupta](#Ria-Gupta)|2020|Social Currency Metric System|Valerio Cosentino, Samantha Venia, Logan|
above line of code will not interlink the [**#Ria-Gupta**](https://github.com/chaoss/website/blob/master/Community/alumni.md#ria-gupta) section on website, it will only add **#Ria-Gupta** to the suffix of the URL

### PR DESCRIPTION
**Pull request by robin025**
fixes some typos

### CODE REVIEW

**Comment by robin025** [comment]
Also Fixed Some typos and mistakes

File: About/About/faq.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
-
 # FAQ
 
-## How do open source community health metrics help me understand whether an open source project is healthy?
+# How do open source community health metrics help me understand whether an open source project is healthy?
```
Comment (GeorgLink) [inline About/About/faq.md:3]: The questions should continue to be a sub-heading with `##`.

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Thanks @robin025!

Please leave the FAQ questions as sub-headings.

I was thinking that we don't need to change the past CHAOSScast event pages, but I don't feel too strongly about that.

File: About/About/faq.md (hunk 1 above)
Comment (robin025) [reply About/About/faq.md]: Done...I made the changes as suggested in the review!!

File: release/release-notes.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Release History
-CHAOSS metrics are released continuously. The regular release is when we update the version number, update the full release notes, and make a big announcement. These releases occur one to two times a year and may correspond with the dates of a CHAOSScon event. Prior to regular release, continuous released metrics go through a comment period of at least 30 days.
+CHAOSS metrics are released continuously. The regular release is when we update the version number, update the full release notes, and make a big announcement. These releases occur one to two times a year and may correspond with the dates of a CHAOSScon event. Before regular release, continuously released metrics go through a comment period of at least 30 days.
```
Comment (GeorgLink) [inline release/release-notes.md:2]: Please undo this change

File: release/metrics-head.md [hunk 3]
```diff
@@ -10,4 +10,4 @@ _To make suggestions or edits to this webpage please visit the [website repo](ht
...
 
-The CHAOSS project recognizes that there are ethical and legal challenges when using the metrics and software provided by the CHAOSS community. **Ethical challenges** exist around protecting community members and empowering them with their personal information. **Legal challenges** exist around GDPR and similar laws or regulations that protect personal information of community members. Particular challenges may arise in the use that is specific to your context.
+The CHAOSS project recognizes that there are ethical and legal challenges when using the metrics and software provided by the CHAOSS community. **Ethical challenges** exist around protecting community members and empowering them with their personal information. **Legal challenges** exist around GDPR and similar laws or regulations that protect the personal information of community members. Particular challenges may arise in the use that is specific to your context.
```
Comment (GeorgLink) [inline release/metrics-head.md:13]: Please undo this change

File: Programs/DIBadging/description.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Diversity & Inclusion Event Badging Form
-Thank you for showing interest in D&I Badging for your event! The Event Badging section of CHAOSS Badging is about measuring inclusivity of different technical events through human reviews.
+Thank you for showing interest in D&I Badging for your event! The Event Badging section of CHAOSS Badging is about measuring the inclusivity of different technical events through human reviews.
```
Comment (GeorgLink) [inline Programs/DIBadging/description.md:2]: Please undo this change

File: CHAOSScon/2020-Shanghai-meetup/speakers.md [hunk 5]
```diff
@@ -71,7 +71,7 @@ Participated in the open source community for more than 6 years. Currently, she
 
 _Director of Sales - Bitergia_
 
-Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg enjoys reading fiction and hot-air ballooning. [@GeorgLink](https://twitter.com/georglink)
+Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years of experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg enjoys reading fiction and hot-air ballooning. [@GeorgLink](https://twitter.com/georglink)
```
Comment (GeorgLink) [inline CHAOSScon/2020-Shanghai-meetup/speakers.md:74]: Please undo this change

File: CHAOSScon/2020-Shanghai-meetup/speakers.md [hunk 6]
```diff
@@ -55,7 +55,7 @@ Xiaoya is currently a postgraduate student of ECNU, major in data science and so
 
 _Co-Founder - Bitergia, InnerSource Commons_
 
-Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests about open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wikimedia or Xen. He has participated as speaker giving details about gender diversity in OpenStack, InnerSource metrics strategy at OSCON, and other metrics-related talks.
+Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests in open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wikimedia or Xen. He has participated as a speaker giving details about gender diversity in OpenStack, InnerSource metrics strategy at OSCON, and other metrics-related talks.
```
Comment (GeorgLink) [inline CHAOSScon/2020-Shanghai-meetup/speakers.md:58]: Please undo this change

File: CHAOSScon/2020-Shanghai-meetup/speakers.md [hunk 7]
```diff
@@ -22,7 +22,7 @@ Wang Wei, researcher and doctoral supervisor of the School of Data Science and E
...
+Ray is the Head of Community at Cube Dev where he is helping to grow the community of contributors to cube.js. Prior to Cube Dev, Ray managed open source communities at GitLab and the Linux Foundation. He has over 15 years of experience in the high-tech industry in roles ranging from a softwaresoftware engineer, product manager, program manager, and team lead at companies such as EDS, Intel, and Medallia. Ray lives in Sunnyvale, CA with his wife and daughter and all three are loyal season ticket holders of the San Jose Earthquakes soccer team. Previously Ray spoke at CHAOSScon, Community Leadership Summit, FOSDEM, GitLab Commit, and Open Source Summit.
```
Comment (GeorgLink) [inline CHAOSScon/2020-Shanghai-meetup/speakers.md:25]: Please undo this change

File: CHAOSScon/2020-Shanghai-meetup/about.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ## About CHAOSS Shanghai Meetup
 
-Meet the CHAOSS community. Learn about metrics and tools used by several open source projects, communities, and engineering teams to track and analyze their development activities, communities health, diversity, risk, and value.
+Meet the CHAOSS community. Learn about metrics and tools used by several open source projects, communities, and engineering teams to track and analyze their development activities, community's health, diversity, risk, and value.
```
Comment (GeorgLink) [inline CHAOSScon/2020-Shanghai-meetup/about.md:3]: Please undo this change

File: CHAOSScon/2019NA/speakers.md [hunk 9]
```diff
@@ -93,7 +93,7 @@ Dawn works on OSS Strategy at Pivotal in London. She has more than 20 years of e
 
 _Director of Sales - Bitergia_
 
-Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg enjoys reading fiction and hot-air ballooning. [@GeorgLink](https://twitter.com/georglink)
+Georg Link is an Open Source Community Strategist. Georg co-founded the Linux Foundation CHAOSS Project to advance analytics and metrics for open source project health. Georg has 13 years of experience as an active contributor to several open source projects and has presented on open source topics at 18 conferences. Georg has an MBA and a Ph.D. in Information Technology. In his spare time, Georg enjoys reading fiction and hot-air ballooning. [@GeorgLink](https://twitter.com/georglink)
```
Comment (GeorgLink) [inline CHAOSScon/2019NA/speakers.md:96]: Please undo this change

File: CHAOSScon/2019NA/speakers.md [hunk 10]
```diff
@@ -24,9 +24,9 @@ _PhD Cand. / DevOps - Polytechnique Montreal_
...
 Before my current experience, I had Worked on interesting software engineering topics for swarms of robots. Moreover, my expertise in cloud computing is an asset, which I bring along. In my thesis, I investigated the phenomena of live migrations and how we can apply both the supervised and reinforced learning technique (Machine learning algorithms) on live migration in the cloud.  
-I was part of the team; team lead on an exciting project at LASSENA Laboratory Montreal, Canada. My main task includes building test bench simulator for micro iBB black boxes for cars in other to reconstruct accident scenario to understand driver's behavior.
+I was part of the team; team lead on an exciting project at LASSENA Laboratory Montreal, Canada. My main task includes building test bench simulator for micro iBB black boxes for cars in other to reconstruct accident scenarios to understand driver's behavior.
```
Comment (GeorgLink) [inline CHAOSScon/2019NA/speakers.md:29]: Please undo this change

File: CHAOSScon/2019NA/speakers.md [hunk 11]
```diff
@@ -24,9 +24,9 @@ _PhD Cand. / DevOps - Polytechnique Montreal_
 
 Ph.D. Candidate.,   Software Engineering, MCIS -- Laboratory, École Polytechnique de Montréal.
 OpenStack Member Level: Foundation Member.
-I am focused on empirical research on ecosystems software releases.  I apply mixed method research as a means to answer both the how and why in my findings, which is beneficiary to both the academic and industrial communities.  I am Using a DevOps approach with machine learning to make crucial decisions. Currently, I am involved with the release team of OpenStack and serving as the PC chair of AI and HPC track, for the submit at Denver 2019. Additionally, I am a Big Data analytics guy.
+I am focused on empirical research on ecosystems software releases.  I apply mixed method research as a means to answer both the how and why in my findings, which is beneficial to both the academic and industrial communities.  I am Using a DevOps approach with machine learning to make crucial decisions. Currently, I am involved with the release team of OpenStack and serving as the PC chair of AI and HPC track, for the submit at Denver 2019. Additionally, I am a Big Data analytics guy.
```
Comment (GeorgLink) [inline CHAOSScon/2019NA/speakers.md:27]: Please undo this change

File: CHAOSScon/2019EU/speakers.md [hunk 12]
```diff
@@ -63,11 +63,11 @@ Session: **A Tale of Metrics Faux Pas: Answers Without Questions**
 
 _Data Analyst - Bitergia_
 
-Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests about open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wikimedia or Xen. He has participated as speaker giving details about gender diversity in OpenStack, InnerSource metrics strategy at OSCON, and other metrics-related talks.
+Daniel Izquierdo is co-founder of Bitergia, a start-up focused on providing metrics and consultancy about open source projects. His main interests in open source are related to the community itself, trying to help community managers, organizations and developers to better understand how the project is performing. He has contributed to several open analytics dashboards such as the OpenStack, Wikimedia or Xen. He has participated as a speaker giving details about gender diversity in OpenStack, InnerSource metrics strategy at OSCON, and other metrics-related talks.
```
Comment (GeorgLink) [inline CHAOSScon/2019EU/speakers.md:66]: Please undo this change

File: CHAOSScon/2019EU/speakers.md [hunk 13]
```diff
@@ -37,11 +37,11 @@ Session: **Your Team’s Open Source Contributions, in One Dashboard**
 
 _Marketing Specialist - Bitergia_
 
-Ana holds a Bachelor degree in Marketing and is currently working at Bitergia, a software development analytics company focused on OpenSource and InnerSource environments.
+Ana holds a Bachelor's degree in Marketing and is currently working at Bitergia, a software development analytics company focused on OpenSource and InnerSource environments.
```
Comment (GeorgLink) [inline CHAOSScon/2019EU/speakers.md:40]: Please undo this change

File: CHAOSScon/2019EU/speakers.md [hunk 14]
```diff
@@ -25,10 +25,10 @@ Session: **Creating a Collection of Panels**
 
 _Open Source Engineer - VMware_
 
-Alex is an Open Source Engineer at VMware Open Source Technology Center, where he works on VMware open source contribution analytics. Before VMware, Alex was a getting his masters in Software Engineering at Polytechnique Montreal, where he researched collaboration and contribution in Open Source communities.
+Alex is an Open Source Engineer at VMware Open Source Technology Center, where he works on VMware open source contribution analytics. Before VMware, Alex was getting his master's in Software Engineering at Polytechnique Montreal, where he researched collaboration and contribution in Open Source communities.
```
Comment (GeorgLink) [inline CHAOSScon/2019EU/speakers.md:28]: Please undo this change

File: CHAOSScon/2019EU/speakers.md [hunk 15]
```diff
@@ -6,7 +6,7 @@
 
 _Researcher and Software Developer - Bitergia_
 
-Software engineer at Bitergia and analytics team member. After finishing his PhD on web page representation for clustering tasks in 2012, he has been working in projects involving data retrieval and analysis from different data sources. He has experience with Java, Spring, Python, MySQL, Apache Solr and ElasticSearch, among others. Apart from implementing metrics with Python and Kibana and according to his close family, he is interested in too many things to write a list here.
+Software engineer at Bitergia and analytics team member. After finishing his PhD on web page representation for clustering tasks in 2012, he has been working on projects involving data retrieval and analysis from different data sources. He has experience with Java, Spring, Python, MySQL, Apache Solr and ElasticSearch, among others. Apart from implementing metrics with Python and Kibana and according to his close family, he is interested in too many things to write a list here.
```
Comment (GeorgLink) [inline CHAOSScon/2019EU/speakers.md:9]: Please undo this change

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Please undo changes to personal bios of CHAOSScon speakers. I want to respect the bios they sent and not alter them (especially not after the event is done).

File: About/About/faq.md (hunk 1 above)
Comment (GeorgLink) [reply About/About/faq.md]: Thank you!

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Please remove the alumni.md changes from this PR and create a separate PR.

It is best practice to have small PRs so that the reviewer can focus on specific changes at a time.

The spelling changes are unrelated to the refactoring of the alumni.md page -- so these should be two separate PRs.

Thanks you!

**Comment by robin025** [comment]
> Thanks @robin025!
> 
> Please leave the FAQ questions as sub-headings.
> 
> I was thinking that we don't need to change the past CHAOSScast event pages, but I don't feel too strongly about that.

@GeorgLink  the thing related to CHAOSScast event pages, I do agree with you but when I was understanding the whole website structure, I suddenly came to these small typos/grammatical mistakes while reading the markdown files and also find some links are not working, so I thought to fix it

**Comment by GeorgLink** [comment]
> @GeorgLink the thing related to CHAOSScast event pages, I do agree with you but when I was understanding the whole website structure, I suddenly came to these small typos/grammatical mistakes while reading the markdown files and also find some links are not working, so I thought to fix it

Okay to fix links and typos. But let's leave the speaker bios untouched if it's stylistic issues.

**Comment by robin025** [comment]
> > @GeorgLink the thing related to CHAOSScast event pages, I do agree with you but when I was understanding the whole website structure, I suddenly came to these small typos/grammatical mistakes while reading the markdown files and also find some links are not working, so I thought to fix it
> 
> Okay to fix links and typos. But let's leave the speaker bios untouched if it's stylistic issues.

Done @GeorgLink reverted changes from speaker bios

**Comment by GeorgLink** [comment]
Thanks @robin025 

Would you be comfortable to squash the commits and rebase them to the current master branch?
If not, that's fine and we can work around it.

**Comment by robin025** [comment]
> Thanks @robin025
> 
> Would you be comfortable to squash the commits and rebase them to the current master branch?
> If not, that's fine and we can work around it.

@GeorgLink  Done.... Squashed all the commits and rebased it to master branch

**Comment by GeorgLink** [comment]
superseded by #590
